File: DofusApi/Models/ForumComment.cs
using System;

namespace DofusApi.Models
{
    public class ForumComment
    {
        public int ID { get; set; }
        public string Content { get; set; }
        public int ForumPostID { get; set; }
        public ForumPost ForumPost { get; set; }
        public DateTime CreatedAt { get; set; }

        public int? UserID { get; set; }
        public User User { get; set; }
    }
}

File: DofusSpa/src/app/_models/forum-comment.ts
import { ForumPost } from "./forum-post";
import { User } from "./user";

export interface ForumComment{
    id : number,
    content : string,
    forumPostID : number,
    forumPost : ForumPost,
    createdAt : Date,
    userID : number,
    user : User
}